joomlarrssb.php

```diff
@@ -184,15 +184,27 @@ public function onContentAfterTitle($context, &$article, &$params, $page)
 		$siteURL = substr(JUri::root(), 0, -1);
 		$itemURL = $siteURL . JRoute::_(ContentHelperRoute::getArticleRoute($article->slug, $article->catid));
 
-		// Get the content and merge in the template; first see if $article->text is defined
-		if (!isset($article->text))
+		// Check if we have an intro text image (First priority)
+		$article->images = json_decode($article->images);
```

Comment:
Should probably check and make sure `$article->images` is actually defined first.  Also, I wouldn't suggest overwriting the article's property with the decoded object, so just make it a local variable.